.changeset/.changeset/silver-wasps-relate.md

```diff
@@ -0,0 +1,5 @@
+---
+"openapi-fetch": patch
+---
+
+Fix throwing middleware example in docs
```

Comment:
Thanks for the changeset, but it’s not needed for docs updates! This bumps the version on npm which we don’t need to do here

Reply:
removed.